delivery/src/on_package_created/main.py

```diff
@@ -11,6 +11,7 @@
 from aws_requests_auth.boto_utils import BotoAWSRequestsAuth # pylint: disable=import-error
 from aws_lambda_powertools.tracing import Tracer # pylint: disable=import-error
 from aws_lambda_powertools.logging.logger import Logger # pylint: disable=import-error
+from ecom.metrics import log_metrics
```

Comment:
Since we're already using the Lambda powertools, should we bump to the latest version (1.0.1) and leverage the [metrics feature](https://awslabs.github.io/aws-lambda-powertools-python/core/metrics/) from it?

If we do, there are other changes that are eluded here (e.g. remove dependencies, remove utility in shared, etc.).

Reply:
Thanks for the suggestion, I made the changes to use the Lambda Powertools Metrics utility